### DISCUSSION TITLE
How add Tailwind in Vue js SFC playground

### DISCUSSION THREAD

**Discussion by kr1ke** [tailwindlabs/tailwindcss#13563] [Help] [answered]
How i can add Tailwind in Vue js Sandbox? - https://play.vuejs.org/

I know exactly how to connect other libraries, for example ElementPlus (https://element-plus.run/) or Vuetify (https://element-plus.run/), but I can't guess how to connect tailwind

**Comment by alex-krasikau** [comment, marked as answer]
Hey @kr1ke!

You can utilize Tailwind CSS CDN for this. While I'm not a Vue expert, this approach could work. Maybe you can find a better way to integrate it.

```vue
<script setup>
import { ref } from 'vue'

const msg = ref('Hello World!')

const tailwindScript = document.createElement('script')
tailwindScript.setAttribute('src', 'https://cdn.tailwindcss.com')
document.head.appendChild(tailwindScript)
</script>

<template>
  <h1 class="text-sm">{{ msg }}</h1>
  <input v-model="msg" />
</template>

```

**Reply by kr1ke** [reply]
thanks!
